src/modules/wallet/utils/urls.ts

```diff
@@ -0,0 +1,4 @@
+let TRANSACTIONS_API_URL = 'https://transactions-api.decentraland.co/v1'
```

Comment:
```suggestion
let TRANSACTIONS_API_URL = 'https://transactions-api.decentraland.org/v1'
```

.co is an internal domain